### PR TITLE
feat(ReactionToaster): Change to animated emojis 🎉

### LINKED ISSUE #10561: Animated reaction emojis
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
The current reaction emojis could be a bit more celebratory. :D 

**Describe the solution you'd like**
Animated emojis! This library could help: https://googlefonts.github.io/noto-emoji-animation/

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #10561

It was agreed that we keep the simple emojis in popover.

### 🖼️ Screenshots / Screencasts

https://github.com/nextcloud/spreed/assets/84044328/d699e336-57ad-4bc0-96fc-c04454f8feaf

### 🚧 Tasks

- [ ] Visual review
- [ ] Code review

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Comment by fancycode** [comment]
Nice! Any chance you could use webp images on browsers that support it to reduce the download sizes? For example the `Concerned.gif` here is 4.4 MBytes while the webp from https://googlefonts.github.io/noto-emoji-animation/ is only 967 KBytes (for the same 512x512 resolution).

**Comment by nickvergessen** [comment]
The size is a bit shocking yeah. Maybe there are smaller versions available? We don't need 512px when we only show them in 26px?

That being said, I guess we should make them a bit bigger? cc @jancborchardt

File: src/components/CallView/shared/ReactionToaster.vue [hunk 1]
```diff
@@ -28,7 +28,13 @@
 			:key="toast.seed"
 			class="toast"
 			:style="styled(toast.name, toast.seed)">
-			<span class="toast__reaction">
+			<img v-if="toast.reactionURL"
+				class="toast__reaction-img"
+				:src="toast.reactionURL"
+				:alt="toast.reaction"
+				width="32"
+				height="32">
```
Comment (Antreesy) [inline src/components/CallView/shared/ReactionToaster.vue:36]:
```suggestion
			<picture v-if="toast.reactionURL">
				<source :srcset="toast.reactionURL.replace('gif', 'webp')" type="image/webp">
				<img class="toast__reaction-img"
					:src="toast.reactionURL"
					:alt="toast.reaction"
					width="32"
					height="32">
			</picture>
```

That's a rough solution, but should do the trick, if we provide both .webp and .gif

**Review by Antreesy — COMMENTED** [review]
Managed to reduce gif's size up to 80% by resizing (https://ezgif.com/resize for example)

And we need to think beforehand of fallbacks, how the emojis roster change may impact

File: src/components/CallView/shared/ReactionToaster.vue (hunk 1 above)
Comment (DorraJaouad) [reply src/components/CallView/shared/ReactionToaster.vue:36]: I will reduce the gif dimensions to 64px which leads to 91% for Concerned emoji for example. This makes gif have less size than webp version. 

p.s: I tried to resize the webp as it is in 512x512 too, but couldn't find a good resizing tool.

**Review by nickvergessen — APPROVED** [review]
Let's get it in? The size can be easily changed afterwards?

**Comment by jancborchardt** [comment]
> That being said, I guess we should make them a bit bigger?

@nickvergessen yeah, they are a bit small currently. We could double the size for sure?

**Pull request merged**